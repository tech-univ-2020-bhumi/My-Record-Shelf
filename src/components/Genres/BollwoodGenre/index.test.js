/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import BollywoodGenre from './index';

describe('The BollywoodGenre component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<BollywoodGenre testId="123" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
