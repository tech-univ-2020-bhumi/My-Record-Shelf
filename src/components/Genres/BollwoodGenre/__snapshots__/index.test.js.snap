// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The BollywoodGenre component should render correctly 1`] = `
<DocumentFragment>
  <div
    data-testid="123"
  >
    <div>
      bollywood
    </div>
    <div />
  </div>
</DocumentFragment>
`;
